contracts/Controller.sol

```diff
@@ -1,12 +1,297 @@
-/**
- * SPDX-License-Identifier: UNLICENSED
- */
-pragma solidity 0.6.10;
+pragma solidity =0.6.10;
+
+import {Ownable} from "./packages/oz/Ownable.sol";
+import {SafeMath} from "./packages/oz/SafeMath.sol";
 
 /**
- *
+ * SPDX-License-Identifier: UNLICENSED
+ * @dev The Controller is...
+ * @author Opyn
  */
 // solhint-disable-next-line no-empty-blocks
-contract Controller {
+contract Controller is Ownable {
+    //WHAT DOES OWNABLE DO?
+    using SafeMath for uint256;
+
+    address internal addressBook;
+
+    bool internal systemPaused;
+    //TODO: should this be internal? also do we need setters and getters for the two above?
+
+    mapping (address  => MarginAccount.Account) internal accounts;
+    mapping (address  => mapping (uint256  => MarginAccount.Vault)) internal vaults;
+    mapping (address => mapping(address => bool)) internal operators;
+
+    event NewOperatorSet(address newOperator);
+
+    modifier isPaused {
+        require(
+            throw
```

Comment:
remove this `throw` otherwise it can't be compiled

Reply:
yeah I think for now remove all the code inside the modifier, just keep the modifier declaration.